plugins/package-managers/node/src/main/kotlin/yarn2/PackageInfo.kt

```diff
@@ -19,38 +19,40 @@
 
 package org.ossreviewtoolkit.plugins.packagemanagers.node.yarn2
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties
-import com.fasterxml.jackson.annotation.JsonProperty
-import com.fasterxml.jackson.module.kotlin.readValues
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeToSequence
 
-import org.ossreviewtoolkit.model.jsonMapper
+private val JSON = Json { ignoreUnknownKeys = true }
 
 internal fun parsePackageInfos(info: String): List<PackageInfo> =
-    jsonMapper.createParser(info).use { parser ->
-        jsonMapper.readValues<PackageInfo>(parser).readAll()
-    }
+    info.byteInputStream().use { JSON.decodeToSequence<PackageInfo>(it) }.toList()
 
+@Serializable
 internal data class PackageInfo(
     val value: String,
     val children: Children
 ) {
-    @JsonIgnoreProperties(ignoreUnknown = true)
+    @Serializable
     data class Children(
-        @JsonProperty("Version")
+        @SerialName("Version")
         val version: String,
-        @JsonProperty("Manifest")
+        @SerialName("Manifest")
         val manifest: Manifest,
-        @JsonProperty("Dependencies")
+        @SerialName("Dependencies")
         val dependencies: List<Dependency> = emptyList()
     )
 
+    @Serializable
     data class Manifest(
-        @JsonProperty("License")
+        @SerialName("License")
         val license: String? = null,
-        @JsonProperty("Homepage")
+        @SerialName("Homepage")
         val homepage: String? = null
     )
 
+    @Serializable
```

Comment:
Just double checking: For this class, serialized properties indeed do not start capitalized?

Reply:
No, they are lower case. 

Just FYI, here is the format output I get for our fun test asset:

```
[
  {
    "value": "@types/node@npm:12.6.8",
    "children": {
      "Version": "12.6.8",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      }
    }
  },
  {
    "value": "asap@npm:2.0.6",
    "children": {
      "Version": "2.0.6",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      }
    }
  },
  {
    "value": "boolbase@npm:1.0.0",
    "children": {
      "Version": "1.0.0",
      "Manifest": {
        "License": "ISC",
        "Homepage": "https://github.com/fb55/boolbase"
      }
    }
  },
  {
    "value": "cheerio@npm:1.0.0-rc.1",
    "children": {
      "Version": "1.0.0-rc.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "css-select@npm:~1.2.0",
          "locator": "css-select@npm:1.2.0"
        },
        {
          "descriptor": "dom-serializer@npm:~0.1.0",
          "locator": "dom-serializer@npm:0.1.1"
        },
        {
          "descriptor": "entities@npm:~1.1.1",
          "locator": "entities@npm:1.1.2"
        },
        {
          "descriptor": "htmlparser2@npm:^3.9.1",
          "locator": "htmlparser2@npm:3.10.1"
        },
        {
          "descriptor": "lodash@npm:^4.15.0",
          "locator": "lodash@npm:4.17.15"
        },
        {
          "descriptor": "parse5@npm:^3.0.1",
          "locator": "parse5@npm:3.0.3"
        }
      ]
    }
  },
  {
    "value": "coffee-script@npm:1.12.7",
    "children": {
      "Version": "1.12.7",
      "Manifest": {
        "License": "MIT",
        "Homepage": "http://coffeescript.org"
      },
      "Exported Binaries": [
        "cake",
        "coffee"
      ]
    }
  },
  {
    "value": "cson-parser@npm:1.3.5",
    "children": {
      "Version": "1.3.5",
      "Manifest": {
        "License": "BSD-3-Clause",
        "Homepage": "https://github.com/groupon/cson-parser"
      },
      "Dependencies": [
        {
          "descriptor": "coffee-script@npm:^1.10.0",
          "locator": "coffee-script@npm:1.12.7"
        }
      ]
    }
  },
  {
    "value": "cson@npm:4.1.0",
    "children": {
      "Version": "4.1.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/cson"
      },
      "Exported Binaries": [
        "cson2json",
        "json2cson"
      ],
      "Dependencies": [
        {
          "descriptor": "coffee-script@npm:^1.12.4",
          "locator": "coffee-script@npm:1.12.7"
        },
        {
          "descriptor": "cson-parser@npm:^1.3.4",
          "locator": "cson-parser@npm:1.3.5"
        },
        {
          "descriptor": "extract-opts@npm:^3.3.1",
          "locator": "extract-opts@npm:3.3.1"
        },
        {
          "descriptor": "requirefresh@npm:^2.1.0",
          "locator": "requirefresh@npm:2.2.0"
        },
        {
          "descriptor": "safefs@npm:^4.1.0",
          "locator": "safefs@npm:4.1.0"
        }
      ]
    }
  },
  {
    "value": "css-select@npm:1.2.0",
    "children": {
      "Version": "1.2.0",
      "Manifest": {
        "License": "BSD-like",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "boolbase@npm:~1.0.0",
          "locator": "boolbase@npm:1.0.0"
        },
        {
          "descriptor": "css-what@npm:2.1",
          "locator": "css-what@npm:2.1.3"
        },
        {
          "descriptor": "domutils@npm:1.5.1",
          "locator": "domutils@npm:1.5.1"
        },
        {
          "descriptor": "nth-check@npm:~1.0.1",
          "locator": "nth-check@npm:1.0.2"
        }
      ]
    }
  },
  {
    "value": "css-what@npm:2.1.3",
    "children": {
      "Version": "2.1.3",
      "Manifest": {
        "License": "BSD-2-Clause",
        "Homepage": null
      }
    }
  },
  {
    "value": "dom-serializer@npm:0.1.1",
    "children": {
      "Version": "0.1.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "domelementtype@npm:^1.3.0",
          "locator": "domelementtype@npm:1.3.1"
        },
        {
          "descriptor": "entities@npm:^1.1.1",
          "locator": "entities@npm:1.1.2"
        }
      ]
    }
  },
  {
    "value": "domelementtype@npm:1.3.1",
    "children": {
      "Version": "1.3.1",
      "Manifest": {
        "License": "BSD-2-Clause",
        "Homepage": null
      }
    }
  },
  {
    "value": "domhandler@npm:2.4.2",
    "children": {
      "Version": "2.4.2",
      "Manifest": {
        "License": "BSD-2-Clause",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "domelementtype@npm:1",
          "locator": "domelementtype@npm:1.3.1"
        }
      ]
    }
  },
  {
    "value": "domutils@npm:1.5.1",
    "children": {
      "Version": "1.5.1",
      "Manifest": {
        "License": null,
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "dom-serializer@npm:0",
          "locator": "dom-serializer@npm:0.1.1"
        },
        {
          "descriptor": "domelementtype@npm:1",
          "locator": "domelementtype@npm:1.3.1"
        }
      ]
    }
  },
  {
    "value": "eachr@npm:3.2.0",
    "children": {
      "Version": "3.2.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/eachr"
      },
      "Dependencies": [
        {
          "descriptor": "editions@npm:^1.1.1",
          "locator": "editions@npm:1.3.4"
        },
        {
          "descriptor": "typechecker@npm:^4.3.0",
          "locator": "typechecker@npm:4.7.0"
        }
      ]
    }
  },
  {
    "value": "editions@npm:1.3.4",
    "children": {
      "Version": "1.3.4",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/editions"
      }
    }
  },
  {
    "value": "editions@npm:2.1.3",
    "children": {
      "Version": "2.1.3",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/editions"
      },
      "Dependencies": [
        {
          "descriptor": "errlop@npm:^1.1.1",
          "locator": "errlop@npm:1.1.1"
        },
        {
          "descriptor": "semver@npm:^5.6.0",
          "locator": "semver@npm:5.7.0"
        }
      ]
    }
  },
  {
    "value": "entities@npm:1.1.2",
    "children": {
      "Version": "1.1.2",
      "Manifest": {
        "License": "BSD-2-Clause",
        "Homepage": null
      }
    }
  },
  {
    "value": "errlop@npm:1.1.1",
    "children": {
      "Version": "1.1.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/errlop"
      },
      "Dependencies": [
        {
          "descriptor": "editions@npm:^2.1.2",
          "locator": "editions@npm:2.1.3"
        }
      ]
    }
  },
  {
    "value": "extract-opts@npm:3.3.1",
    "children": {
      "Version": "3.3.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/extract-opts"
      },
      "Dependencies": [
        {
          "descriptor": "eachr@npm:^3.2.0",
          "locator": "eachr@npm:3.2.0"
        },
        {
          "descriptor": "editions@npm:^1.1.1",
          "locator": "editions@npm:1.3.4"
        },
        {
          "descriptor": "typechecker@npm:^4.3.0",
          "locator": "typechecker@npm:4.7.0"
        }
      ]
    }
  },
  {
    "value": "graceful-fs@npm:4.2.0",
    "children": {
      "Version": "4.2.0",
      "Manifest": {
        "License": "ISC",
        "Homepage": null
      }
    }
  },
  {
    "value": "htmlparser2@npm:3.10.1",
    "children": {
      "Version": "3.10.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "domelementtype@npm:^1.3.1",
          "locator": "domelementtype@npm:1.3.1"
        },
        {
          "descriptor": "domhandler@npm:^2.3.0",
          "locator": "domhandler@npm:2.4.2"
        },
        {
          "descriptor": "domutils@npm:^1.5.1",
          "locator": "domutils@npm:1.5.1"
        },
        {
          "descriptor": "entities@npm:^1.1.1",
          "locator": "entities@npm:1.1.2"
        },
        {
          "descriptor": "inherits@npm:^2.0.1",
          "locator": "inherits@npm:2.0.4"
        },
        {
          "descriptor": "readable-stream@npm:^3.1.1",
          "locator": "readable-stream@npm:3.4.0"
        }
      ]
    }
  },
  {
    "value": "inherits@npm:2.0.4",
    "children": {
      "Version": "2.0.4",
      "Manifest": {
        "License": "ISC",
        "Homepage": null
      }
    }
  },
  {
    "value": "lodash@npm:4.17.15",
    "children": {
      "Version": "4.17.15",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://lodash.com/"
      }
    }
  },
  {
    "value": "long@npm:3.2.0",
    "children": {
      "Version": "3.2.0",
      "Manifest": {
        "License": "Apache-2.0",
        "Homepage": null
      }
    }
  },
  {
    "value": "npm-yarn@workspace:.",
    "children": {
      "Version": "1.0.0",
      "Manifest": {
        "License": "Apache-2.0",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "cheerio@npm:1.0.0-rc.1",
          "locator": "cheerio@npm:1.0.0-rc.1"
        },
        {
          "descriptor": "cson@npm:~4.1.0",
          "locator": "cson@npm:4.1.0"
        },
        {
          "descriptor": "long@npm:^3.2.0",
          "locator": "long@npm:3.2.0"
        },
        {
          "descriptor": "promise@npm:~7.3.1",
          "locator": "promise@npm:7.3.1"
        }
      ]
    }
  },
  {
    "value": "nth-check@npm:1.0.2",
    "children": {
      "Version": "1.0.2",
      "Manifest": {
        "License": "BSD-2-Clause",
        "Homepage": "https://github.com/fb55/nth-check"
      },
      "Dependencies": [
        {
          "descriptor": "boolbase@npm:~1.0.0",
          "locator": "boolbase@npm:1.0.0"
        }
      ]
    }
  },
  {
    "value": "parse5@npm:3.0.3",
    "children": {
      "Version": "3.0.3",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/inikulin/parse5"
      },
      "Dependencies": [
        {
          "descriptor": "@types/node@npm:*",
          "locator": "@types/node@npm:12.6.8"
        }
      ]
    }
  },
  {
    "value": "promise@npm:7.3.1",
    "children": {
      "Version": "7.3.1",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "asap@npm:~2.0.3",
          "locator": "asap@npm:2.0.6"
        }
      ]
    }
  },
  {
    "value": "readable-stream@npm:3.4.0",
    "children": {
      "Version": "3.4.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": null
      },
      "Dependencies": [
        {
          "descriptor": "inherits@npm:^2.0.3",
          "locator": "inherits@npm:2.0.4"
        },
        {
          "descriptor": "string_decoder@npm:^1.1.1",
          "locator": "string_decoder@npm:1.2.0"
        },
        {
          "descriptor": "util-deprecate@npm:^1.0.1",
          "locator": "util-deprecate@npm:1.0.2"
        }
      ]
    }
  },
  {
    "value": "requirefresh@npm:2.2.0",
    "children": {
      "Version": "2.2.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/requirefresh"
      },
      "Dependencies": [
        {
          "descriptor": "editions@npm:^2.1.3",
          "locator": "editions@npm:2.1.3"
        }
      ]
    }
  },
  {
    "value": "safe-buffer@npm:5.1.2",
    "children": {
      "Version": "5.1.2",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/feross/safe-buffer"
      }
    }
  },
  {
    "value": "safefs@npm:4.1.0",
    "children": {
      "Version": "4.1.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/safefs"
      },
      "Dependencies": [
        {
          "descriptor": "editions@npm:^1.1.1",
          "locator": "editions@npm:1.3.4"
        },
        {
          "descriptor": "graceful-fs@npm:^4.1.4",
          "locator": "graceful-fs@npm:4.2.0"
        }
      ]
    }
  },
  {
    "value": "semver@npm:5.7.0",
    "children": {
      "Version": "5.7.0",
      "Manifest": {
        "License": "ISC",
        "Homepage": null
      },
      "Exported Binaries": [
        "semver"
      ]
    }
  },
  {
    "value": "string_decoder@npm:1.2.0",
    "children": {
      "Version": "1.2.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/nodejs/string_decoder"
      },
      "Dependencies": [
        {
          "descriptor": "safe-buffer@npm:~5.1.0",
          "locator": "safe-buffer@npm:5.1.2"
        }
      ]
    }
  },
  {
    "value": "typechecker@npm:4.7.0",
    "children": {
      "Version": "4.7.0",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/bevry/typechecker"
      },
      "Dependencies": [
        {
          "descriptor": "editions@npm:^2.1.0",
          "locator": "editions@npm:2.1.3"
        }
      ]
    }
  },
  {
    "value": "util-deprecate@npm:1.0.2",
    "children": {
      "Version": "1.0.2",
      "Manifest": {
        "License": "MIT",
        "Homepage": "https://github.com/TooTallNate/util-deprecate"
      }
    }
  }
]
```

Reply:
> they are lower case.

Weird, but thanks for confirming.